### PR TITLE
Revert "Added new event: portal storm."

### PR DESCRIPTION
**Pull request by Thunder12345**
Reverts tgstation/-tg-station#15052
>Spawning Nar-Sie twice in a single gang round
This is hell. Pure hell.

### CODE REVIEW

File: code/modules/events/portal_storm.dm [hunk 1]
```diff
@@ -1,133 +0,0 @@
...
-
-/datum/round_event/portal_storm/syndicate_shocktroop
-	boss_types = list(/mob/living/simple_animal/hostile/syndicate/melee/space/stormtrooper = 2)
-	hostile_types = list(/mob/living/simple_animal/hostile/syndicate/melee/space/noloot = 8,\
-						/mob/living/simple_animal/hostile/syndicate/ranged/space/noloot = 2)
-
-/datum/round_event_control/portal_storm_narsie
-	name = "Portal Storm: Nar-sie"
-	typepath = /datum/round_event/portal_storm/portal_storm_narsie
-	weight = -1
```
Comment (Incoming5643) [inline code/modules/events/portal_storm.dm]: Well there's your problem, -1 means ALWAYS RUN

**Comment by KorPhaeron** [comment]
What if we fixed the bug or commented out the event or changed the event weight instead of undoing the whole thing

File: code/modules/events/portal_storm.dm (hunk 1 above)
Comment (KazeEspada) [inline code/modules/events/portal_storm.dm]: The event subsystem is undocumented garbage, and this is the kinda error that supposed to be caught by a maintainer.

**Comment by KorPhaeron** [comment]
It's a beta they'll fix it

**Comment by PKPenguin321** [comment]
holy fuck my sides

**Comment by tkdrg** [comment]
I told him to use -1 I thought it meant never run not always
0 doesn't mean never run either so ????

**Pull request closed without merging**